src/drivers/pwm_out_sim/pwm_out_sim.cpp

```diff
@@ -512,6 +516,19 @@ PWMSim::task_main()
 				}
 			}
 
+			/* overwrite outputs in case of force_failsafe */
+			if (_failsafe) {
+				for (size_t i = 0; i < num_outputs; i++) {
+					outputs.output[i] = 600;
```

Comment:
I chose 600 as a magic for PWM Simulator failsafe as it doesn't read values from settings (might need to add this instead) and I needed some distinguishable value from disarmed to easily debug failsafe and not confuse with other states.